explainer.md

```diff
@@ -196,17 +197,31 @@ A single-certificate deployment model forces subscribers to find a single certif
 
 A subscriber may obtain certificate paths from multiple CAs for redundancy in the face of future CA compromises. If one CA is compromised and removed from newer relying parties, the TLS server software will transparently serve the other one.
 
+### Public Key Pinning
+
+To reduce the risk of attacks from misissued certificates, TLS clients sometimes employ [public key pinning](https://www.rfc-editor.org/rfc/rfc7469.html). This enforces that one of some set of public keys appear in the final certificate path. This effectively reduces a client's trust anchor list to a subset.
+
+As above, such a client decision constrains how the server evolves over time. As other clients in the PKI evolve, the pinning clients limit the server to satisfy both the pinning constraint and newer constraints in the PKI. This can lead to conflicts if, for example, the pinned CA is distrusted by a newer client. The server is then forced to either break the pinning clients, or break the newer ones.
+
+Trust anchor negotiation relieves this conflict. The server can select a certificate from the pinned CA with the pinning client, and another CA with newer clients. The server must decide whether to continue to obtaining certificates from pinned CA, or drop support for those pinning clients, but negotiation decouples this decision from the newer clients.
+
+For this to work, the pinning client must accurately negotiate its reduced trust anchor list. TLS Trust Expressions is not as well-suited for this scenario, as it cannot as efficiently represent arbitrary trust store subsets. However, the same supporting infrastructure can be used with the existing `certificate_authorities` extension. While size often makes `certificate_authorities` impractical, a pinning client's reduced trust anchor list is small. Alternatively, the TLS Trust Anchor Identifiers option can handle this case efficiently than `certificate_authorities`.
```

Comment:
handle this case more efficiently

Reply:
and perhaps reword "the TLS Trust Anchor Identifiers option" as just TLS Trust Anchor Identifiers (to mimic how you use TLS Trust Expressions)

Reply:
Rephrased a bit. Put the TAI option first, which seemed more natural anyway.